lib/src/buttons/help_button.dart

```diff
@@ -170,24 +168,25 @@ class _HelpButtonState extends State<HelpButton>
             opacity: _opacityAnimation,
             child: DecoratedBox(
               decoration: BoxDecoration(
-                  shape: BoxShape.circle,
-                  color: !enabled
-                      ? CupertinoDynamicColor.resolve(disabledColor!, context)
-                      : backgroundColor,
-                  boxShadow: [
-                    BoxShadow(
-                      color: Color.fromRGBO(0, 0, 0, 0.1),
-                      offset: Offset(-0.1, -0.1),
-                    ),
-                    BoxShadow(
-                      color: Color.fromRGBO(0, 0, 0, 0.1),
-                      offset: Offset(0.1, 0.1),
-                    ),
-                    BoxShadow(
-                      color: CupertinoColors.tertiarySystemFill,
-                      offset: Offset(0, 0),
-                    ),
-                  ],),
+                shape: BoxShape.circle,
+                color: !enabled
+                    ? CupertinoDynamicColor.resolve(disabledColor!, context)
```

Comment:
Use `DynamicColorX.macosResolve` instead of `CupertinoDynamicColor.resolve`

Reply:
Can you tell me the reason behind using DynamicColorX.macosResolve instead of CupertinoDynamicColor.resolve ?

Reply:
`CupertinoDynamicColor.resolve` uses the `CupertinoTheme` to resolve the colors, but `DynamicColorX.macosResolve` uses `MacosTheme` to resolve them.